bin/hook-reference/index.js

```diff
@@ -33,7 +43,7 @@ const writeJSONFile = async ( data ) => {
 console.log( chalk.green( 'Preparing Hook Reference data file' ) );
 console.log( '\n' );
 
-getFilePaths( 'client' )
+getAllFilePaths( [ 'client', 'packages/navigation/src' ] )
```

Comment:
Curious is there's a reason to not include all of `packages/`.  Is this purely for performance?

Reply:
Good question, I must have been trying to avoid searching through build files and node_modules. I added a check to skip those directories in https://github.com/woocommerce/woocommerce-admin/pull/6833/commits/84e4d1e672d8beb9aa9b9622a712bf7dda112efa. Now it looks through all things in `packages/`